### ISSUE TITLE
[T-SQL] CREATE LOGIN not supported

### ISSUE THREAD

**Issue by flipflop97** [sqlfluff/sqlfluff#5456]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

The syntax for `CREATE LOGIN` is not supported, sqlfluff throws an error when reading it

### Expected Behaviour

CREATE LOGIN should be recognized and parsed

https://learn.microsoft.com/en-us/sql/t-sql/statements/create-login-transact-sql

### Observed Behaviour

```
L:   1 | P:   1 |  PRS | Line 1, Position 1: Found unparsable section: 'CREATE
                       | LOGIN login
```

### How to reproduce

```bash
echo 'CREATE LOGIN login' | sqlfluff lint --dialect tsql -
```

### Dialect

Transact-SQL

### Version

`sqlfluff, version 2.3.5`

`Python 3.11.6`

### Configuration

Default TSQL

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
